CONTRIBUTING.md

```diff
@@ -28,13 +28,21 @@ We default to using notebook-based tutorials.
 
 ## Dev Setup
 
-The only required dev setup is installing [`tox`](https://tox.readthedocs.io).
-For example, if you use `pip`:
+For dev setup, you  will need to install [`tox`](https://tox.readthedocs.io), and set up a virtualenv with all the requirements.
+For example, if you use `pip`, and want to work on the `spam` tutorial:
 
 ```bash
 python3 -m pip install -U 'tox>=3.13.0,<4.0.0'
+python3 -m pip install --upgrade virtualenv
+virtualenv -p python3 .env
+source .env/bin/activate
+
+pip3 install -r requirements.txt
```

Comment:
here too!!

Reply:
@brahmaneya

Reply:
Done. I thought the change was meant for before switching to the venv.